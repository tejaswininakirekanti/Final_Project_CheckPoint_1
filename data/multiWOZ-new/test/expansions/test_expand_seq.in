i would like to find a train that leaves monday and arrives by 19:00 .
i would like to depart from stevenage after 19:45 .
i am looking for a train leaving on tuesday please .
i would just like to arrive by 16:45 please .
i would like a train to cambridge that arrives by 11:30 .
i am departing from ely .
i am head to leicester from cambridge on saturday .
i will be leaving from london kings cross and need to leave after 17:15 .
i am going from cambridge to the stansted airport .
i am looking for a train please going to cambridge .
i am looking for a train leaving from london liverpool street after 20:00 on friday .
i am looking for a train going to cambridge please .
i am looking for a train from cambridge to the stansted airport .
i would like a train that can get me from cambridge to birmingham new street on monday by 12:45 .
i am looking for a train out of cambridge on saturday .
i am looking for a train going to ely please .
i would like it to be for saturday .
i would like a train to ely that leaves wednesday .
i will be leaving on friday after 12:45 .
i would like to leave from broxbourne sometime after 12:30 .
i would like a train to london liverpool street that leaves after 10:00 .
i am looking for a train departing from broxbourne .
i would like to find a train leaving on friday to cambridge .
i am looking for a train from cambridge .
i would like to travel on thursday
i am looking for a train going to cambridge .
i would like to find a train to peterborough .
i will will be leaving from london liverpool street .
i am looking for a train going to broxbourne please .
i am looking for a train that will be going to london kings cross on wednesday .
i am at bishops stortford and would like to go to cambridge .
i will be going from cambridge to broxbourne .
i am looking for a train to take 5me to cambridge .
i will be going to peterborough from cambridge .
i will be leaving stansted airport friday to head toward cambridge .
i am heading to broxbourne .
i would like to go after 10:00 .
i will be traveling on tuesday .
i will be coming from london kings cross
i am leaving london liverpool street on sunday after 20:30 .
i would like to leave on friday and arrive by 14:45 .
i am going from stevenage to cambridge .
i would like to arrive by 13:45 .
i am traveling from cambridge to the london liverpool street .
i would like to find a train tuesday departing from leicester .
i would like to find a train from leiceter on saturday .
i am looking for a train from norwich to cambridge .
i will be leaving from cambridge and i want to go to stansted airport .
i am going to cambridge
i am looking for a train departing from stevenage .