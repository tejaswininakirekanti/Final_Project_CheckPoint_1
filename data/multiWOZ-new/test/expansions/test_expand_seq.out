O O O O O O O O O B-day O O O B-arrive O
O O O O O O B-depart O B-leave O
O O O O O O O O B-day O O
O O O O O O O B-arrive O O
O O O O O O B-dest O O O B-arrive O
O O O O B-depart O
O O O O B-dest O B-depart O B-day O
O O O O O B-dest I-dest I-dest O O O O O B-leave O
O O O O B-depart O O B-dest I-dest O
O O O O O O O O O B-dest O
O O O O O O O O B-depart I-depart I-depart O B-leave O B-day O
O O O O O O O O B-dest O O
O O O O O O O B-depart O O B-dest I-dest O
O O O O O O O O O O B-depart O B-dest I-dest I-dest O B-day O B-arrive O
O O O O O O O O B-depart O B-day O
O O O O O O O O B-dest O O
O O O O O O O B-day O
O O O O O O B-dest O O B-day O
O O O O O B-day O B-leave O
O O O O O O B-depart O O B-leave O
O O O O O O B-dest I-dest I-dest O O O B-leave O
O O O O O O O O B-depart O
O O O O O O O O O B-day O B-dest O
O O O O O O O B-depart O
O O O O O O B-day
O O O O O O O O B-dest O
O O O O O O O O B-dest O
O O O O O O B-depart I-depart I-depart O
O O O O O O O O B-dest O O
O O O O O O O O O O O B-dest I-dest I-dest O B-day O
O O O B-depart I-depart O O O O O O B-dest O
O O O O O B-depart O B-dest O
O O O O O O O O O O B-dest O
O O O O O B-dest O B-depart O
O O O O B-depart I-depart B-day O O O B-dest O
O O O O B-dest O
O O O O O O B-leave O
O O O O O B-day O
O O O O O B-depart I-depart I-depart
O O O B-depart I-depart I-depart O B-day O B-leave O
O O O O O O B-day O O O B-arrive O
O O O O B-depart O B-dest O
O O O O O O B-arrive O
O O O O B-depart O O B-dest I-dest I-dest O
O O O O O O O B-day O O B-depart O
O O O O O O O O O O B-day O
O O O O O O O B-depart O B-dest O
O O O O O B-depart O O O O O O B-dest I-dest O
O O O O B-dest
O O O O O O O O B-depart O